jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/Bridge.kt

```diff
@@ -249,7 +255,27 @@ class Bridge @JvmOverloads internal constructor(
 
     fun endpointRemoved() = endpointsRemoved(1)
     fun endpointsRemoved(count: Int) {
-        endpoints.addAndGet(-count);
+        endpoints.addAndGet(-count)
+    }
+    fun endpointRequestedRestart() {
+        endpointRequestRestartRate.update(1)
+
+        if (config.endpointRestartRequestEnabled) {
+            val restartCount = endpointRequestRestartRate.getAccumulatedCount()
+            val endpoints = endpoints.get()
+            if (endpoints > config.endpointRestartRequestMinEndpoints &&
+                restartCount > endpoints * config.endpointRestartRequestThreshold
+            ) {
+                // Reset the timeout regardless of the previous state, but only log if the state changed.
```

Comment:
This comment says "Reset the timeout", but does it?

Reply:
Yes, by setting `isOperational = false`